### ISSUE TITLE
Inconsistent behavior of jq with no options when standard output is redirected

### ISSUE THREAD

**Issue by wtanaka** [jqlang/jq#1243]
```
~/Desktop % cat test.json
{}
```

Example 1:

```
~/Desktop % cat test.json | jq
{}
```

Example 2:

```
~/Desktop % cat test.json | jq | cat
jq - commandline JSON processor [version 1.5-1-a5b5cbe]
Usage: jq [options] <jq filter> [file...]

    jq is a tool for processing JSON inputs, applying the
    given filter to its JSON text inputs and producing the
    filter's results as JSON on standard output.
    The simplest filter is ., which is the identity filter,
    copying jq's input to its output unmodified (except for
    formatting).
    For more advanced filters see the jq(1) manpage ("man jq")
    and/or https://stedolan.github.io/jq

    Some of the options include:
     -c     compact instead of pretty-printed output;
     -n     use `null` as the single input value;
     -e     set the exit status code based on the output;
     -s     read (slurp) all inputs into an array; apply filter to it;
     -r     output raw strings, not JSON texts;
     -R     read raw strings, not JSON texts;
     -C     colorize JSON;
     -M     monochrome (don't colorize JSON);
     -S     sort keys of objects on output;
     --tab  use tabs for indentation;
     --arg a v  set variable $a to value <v>;
     --argjson a v  set variable $a to JSON value <v>;
     --slurpfile a f    set variable $a to an array of JSON texts read from <f>;
    See the manpage for more options.
```

Expected: Example 1 and Example 2 behave the same way

**Comment by pkoppstein** [comment]
Please note that this issue was addressed some time ago, but there has been a delay in releasing version 1.6 of jq. In the meantime, the fix is available in the "development" version of jq (aka "master"):

```
$ cat empty.json | jq
{}
$ cat empty.json | jq | cat
{}
$ jq --version
jq-1.5rc2-150-g1740fd0
```

**Comment by itchyny** [comment]
This bug was already fixed and released in [1.6](https://github.com/jqlang/jq/releases/tag/jq-1.6) in Nov 2018, so closing this issue.
